docs/dashboards/kubernetes-dashboard.asciidoc

```diff
@@ -0,0 +1,73 @@
+
+**Support Matrix**: This feature is currently available on GKE and EKS using Linux hosts and Kubernetes versions that match the following specifications:
+|=====================
+| | **Kubernetes versions** | **Container runtime** | **Nose OSes**
+|**EKS**| 1.18; 1.19; 1.20; 1.21| Docker | Amazon Linux 2, Bottlerocket OS
+|**GKE**| Regular (default channel): 1.21 and 1.22; Stable: 1.20 and 1.21; Rapid: 1.22 and 1.23 | containerd | Container-optimized OS (COS), Ubuntu
+|=====================
+You first need to download the daemonset manifest .yaml, then modify it to include your {fleet} URL and Enrollment Token before you deploy it to the clusters you want to monitor. After that, data will start to appear in the Kubernetes dashboard.
+
+[discrete]
+== Download and modify the daemonset manifest
+The daemonset integrates Elastic Endpoint into your Kubernetes cluster. The Elastic Agent is enrolled to a running Fleet Server using the `FLEET_URL` parameter, and connected to a specific Elastic Agent policy using the `FLEET_ENROLLMENT_TOKEN`.
+
+1. Download the daemonset manifest http://github.com/elastic/placeholder[here] for GKE or http://github.com/elastic/placeholder[here] for EKS.
+2. Fill in the manifest's `FLEET_URL` field with your {fleet} server's `Host URL`. To find it, go to **Kibana > Management > {fleet} > Settings**. For more information, refer to {fleet-guide}/fleet-settings.html[Fleet UI settings].
+3. Fill in the manifest's `FLEET_ENROLLMENT_TOKEN` field with a Fleet enrollment token. To find one, go to **Kibana > Management > {fleet} > Enrollment tokens**. For more information, refer to {fleet-guide}/fleet-enrollment-tokens.html[Fleet enrollment tokens].
+
+IMPORTANT: The Elastic Agent policy you use must include the {endpoint-cloud-sec} integration, and must have `Include session data` set to `true`. To modify this setting, go to **Security > Manage > Policies**, select your policy, and find `Include session data` near the bottom of the `Policy settings` tab:
+
+image::session-data-config.png[the "Enable session data" setting on the Policy settings page]
+
+
+[discrete]
+== Apply the modified manifest to your cluster or clusters
+First, to ensure you install Endpoint on the desired k8s cluster(s), use `kubectl config use-context <name-of-context>` to set the default context. You can repeat the following process for multiple contexts.
+
+NOTE: To check which contexts exist, use `kubectl config get-contexts` to list them from your local kubectl config file. An asterisk, if present, indicates the current default context.
+
+To apply the manifest to a cluster, run for example: `kubectl apply -f eks-elastic-security.yaml` for EKS, or `kubectl apply -f gke-elastic-security.yaml` for GKE.
```

Comment:
This would be good as bullets, since there are two options here:
```suggestion
To apply the manifest to a cluster, run for example: 
- EKS: `kubectl apply -f eks-elastic-security.yaml`
- GKE: `kubectl apply -f gke-elastic-security.yaml`
```